praw/endpoints.py

```diff
@@ -138,6 +138,7 @@
     "quarantine_opt_in":       "api/quarantine_optin",
     "quarantine_opt_out":      "api/quarantine_optout",
     "read_message":            "api/read_message/",
+    "read_all_messages":       "api/read_all_messages",
```

Comment:
```suggestion
    "read_all_messages":       "api/read_all_messages",
    "read_message":            "api/read_message/",
```
Can we sort this too?